afterDownloadingIndexHtml/afterDownloadingIndexHtml.md

```diff
@@ -0,0 +1,58 @@
+# After Downloading `index.html`
+
+This documents what happens after the user loads `index.html` of `hollowverse.com`
+
+## Notable Person page (e.g. /Tom_Hanks)
+
+```mermaid
+sequenceDiagram
+  participant browser as Browser
+  participant api as api hollowverse com
+  participant aurora as AWS Aurora
+  participant photos as photos hollowverse com
+  participant s3 as AWS S3 bucket
```

Comment:
photos.hollowverse.com is an alias for a CloudFront distrubition put in front of the S3 bucket. We may want to add CF in between.